multiqc/utils/search_patterns.yaml

```diff
@@ -55,6 +55,27 @@ cutadapt:
     contents: 'This is cutadapt'
     # contents: 'cutadapt version' # Use this instead if using very old versions of cutadapt (eg. v1.2)
     shared: true
+deeptools/bamPEFragmentSize:
+    contents: '	Frag. Sampled	Frag. Len. Min.	Frag. Len. 1st. Qu.	Frag. Len. Mean	Frag. Len. Median	Frag. Len. 3rd Qu.'
+    num_lines: 1
+deeptools/estimateReadFiltering:
+    contents: 'Sample	Total Reads	Mapped Reads	Alignments in blacklisted regions	Estimated mapped reads'
+    num_lines: 1
+deeptools/plotCoverageStdout:
+    contents: 'sample	mean	std	min	25%	50%	75%	max'
+    num_lines: 1
+deeptools/plotCoverageOutRawCounts:
+    contents: "#'chr'	'start'	'end'	'"
```

Comment:
ooh, is this really the most specific string we can use? I could imagine a lot of other outputs starting with the same...

Reply:
Maybe I'll just add a more useful header line to the output real quick... :)

Reply:
Aha, now you're talking! 😉  I'm a big fan of adding tool name and version as a header line to all output file names. If you have the version name too then we can plug it into a future feature that grabs these automatically where possible (see https://github.com/ewels/MultiQC/pull/536)